bucketlist/src/main/kotlin/io/aexp/bucketlist/BucketListClient.kt

```diff
@@ -34,6 +35,12 @@ interface BucketListClient {
             Observable<PagedResponse<PullRequestCommit>>
 
     /**
+     * @return observable of PR diff response
+     */
+    fun getPrDiff(projectKey: String, repoSlug: String, prId: Long, contextLines: Int, whitespace: Boolean,
```

Comment:
It's not really clear to me what contextLines, whitespace or withComments arguments are doing.